.github/workflows/pr-push.yml

```diff
@@ -0,0 +1,32 @@
+name: Node CI
+
+on:
+  push:
+    branches:
+      - gh-pages
+  pull_request: {}
+
+jobs:
+  ci:
+    name: Validate and Publish
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 12
+      - name: Do spec validation
+        env:
+          GH_USER: ${{ secrets.GH_USER }}
+          GH_TOKEN: ${{ secrets.GH_TOKEN }}
+        run: npx respec-validator --gh-user=$GH_USER --gh-token=$GH_TOKEN index.html
```

Comment:
I think you need to install ReSpec unfortunately.

Reply:
respec is a dependency of respec-validator, so it should've been installed with npx, right?
let me try locally also

Reply:
Interesting

Reply:
Glad we did a test run

Reply:
Pfff... whatever. :shipit: